racetrack_client/racetrack_client/main.py

```diff
@@ -281,6 +281,28 @@ def _list_installed_plugins(
         list_installed_plugins(remote)
 
 
+@cli_plugin.command('download-installed-plugins')
```

Comment:
Do we really need this option? I think it makes it a bit more complicated unnecessarily.